src/python/bot/fuzzers/mutator_plugin.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Getting and using custom mutator plugins for libFuzzer."""
+
+import os
+
+from base import utils
+from google_cloud_utils import gsutil
+from google_cloud_utils import storage
+from system import archive
+from system import environment
+from system import shell
+
+MUTATOR_PLUGINS_BUCKET_ENV_VAR = 'MUTATOR_PLUGINS_BUCKET'
```

Comment:
this seems like an unnecessary indirection. just use 'MUTATOR_PLUGINS_BUCKET'  ?